### PR TITLE
Sound library update

### PR DESCRIPTION
**Pull request by ericrosenbaum**
### Resolves

Resolves https://github.com/LLK/scratch-gui/issues/2710
Resolves https://github.com/LLK/scratch-gui/issues/2844

### Proposed Changes

- Add 14 new music loops to the sound library! They are all under either public domain, CC0, or CC-BY licenses, and come from freesound.org, freemusicarchive.org, incompetech.com and archive.org.

Air Hockey Saloon - [Chris Zabriskie](http://freemusicarchive.org/music/Chris_Zabriskie/Vendaface/05_-_Air_Hockey_Saloon)
Bossa Bossa - [Kevin MacLeod](https://incompetech.com/music/royalty-free/index.html?isrc=USUAN1600055)
Darkling - [Kevin MacLeod](https://incompetech.com/music/royalty-free/index.html?isrc=USUAN1700050)
Energetic Dance - [furbyguy](http://freesound.org/people/furbyguy/sounds/331881/)
Dance Sitar - [Nina Paley](https://archive.org/details/SitaSingsTheBluesSoundtrack/sstb-07-ganpati.wav)
Variations 1 - [Kimiko Ishizaka](http://freemusicarchive.org/music/Kimiko_Ishizaka/The_Open_Goldberg_Variations/KIMIKO_ISHIZAKA_-_Goldberg_Variations_BWV_988_-_02_-_Variatio_1_a_1_Clav__44k-24b)
Inhale - [Peter Rudenko](http://freemusicarchive.org/music/Peter_Rudenko/Incomplete/04_-_Peter_Rudenko_-_Inhale)
Melodic Dubstep - [Orboscope](http://freesound.org/people/Orboscope/sounds/377144/)
Mystery - [tyops](https://freesound.org/people/tyops/sounds/402273/)
Nephilim - [Flick3r](http://freesound.org/people/Flick3r/sounds/48481/)
Pookatori and Friends - [Kevin MacLeod](https://incompetech.com/music/royalty-free/index.html?isrc=USUAN1700002)
Reggae Nine Mile - [Nueve Millas](https://archive.org/details/NueveMillas-NineMilesfromKingston/02_Nueve_Millas_-_Regrezion.mp3)
Send for the Horses - [Kevin MacLeod](http://incompetech.com/music/royalty-free/index.html?isrc=USUAN1700028)
Trapped - [Kellee Maize](http://freemusicarchive.org/music/Kellee_Maize/Integration/Trapped)

- Also, remove the "-b" from "Odesong-b"
- Rename the sounds "Connect" and "Disconnect" to "Start" and "Finish"

### Reason for Changes

This is progress toward a more diverse and higher quality library of music loops for people to use in their projects (and a couple of minor fixes).

### CODE REVIEW

**Comment by Mr-Dave2** [comment]
Do you take requests?

I think the "Happy Birthday to you" song could stand to be upgraded.  It's just called "Birthday" in the Scratch 3.0 sound library (and is called Birthday Bells in Scratch 2.0)

The song and Lyrics are now in the public domain.  You can list it (or a replacement) by its full title which would make it easier to find.

This song is very popular and I believe that beginners would use it for inspiration in their projects if a better version was included with Scratch.  (The current version is just the melody play on xylophone).

I have a "test case" here of a version that I arranged and played on my keyboard. https://scratch.mit.edu/projects/248022990/

Also, and I'm sure you know this, but not all recordings will be in the public domain.  Individual arrangements and performances might still be copyrighted.

**Comment by ericrosenbaum** [comment]
@towerofnix ah! true, it's from a movie called "Sita Sings the Blues," but I called the track that because it features the sound of a stringed instrument called a [sitar](https://en.wikipedia.org/wiki/Sitar).

**Comment by towerofnix** [comment]
@ericrosenbaum today I learned! - I never heard of the sitar before. Neat :)

File: src/lib/libraries/sounds.json [hunk 1]
```diff
@@ -2858,6 +2987,18 @@
             "bird"
         ]
     },
+    {
+        "name": "Send For The Horses",
```
Comment (thisandagain) [inline src/lib/libraries/sounds.json]: Hmm. "Movie 2"?

File: src/lib/libraries/sounds.json [hunk 2]
```diff
@@ -108,6 +108,19 @@
         "format": "",
         "tags": []
     },
+    {
+        "name": "Air Hockey Saloon",
```
[the remark this answers is not in the document]
Comment (thisandagain) [reply src/lib/libraries/sounds.json]: "Movie 1"

**Comment by ericrosenbaum** [comment]
@Mr-Dave2 thanks for the suggestion! I agree that our happy birthday could use an upgrade :)

**Pull request merged**